backend/essays/module/infer.py

```
@@ -0,0 +1,287 @@
+# --------------------------------------------------------------------------- #
+def build_polylines(result: dict) -> list[dict]:
+    """Convert the wrapper output to the app's polyline JSON schema.
+
+    centerlines are (row, col) px; the app stores (x, y) = (col, row).
+    The 32-d embedding samples are base64-encoded float16 so the tracker /
+    kymograph endpoints can consume the file unchanged.
+    """
+    centerlines = result["centerlines_rc"]
+    embeddings = result["embedding_samples"]
+    polylines = []
+    for i, (cl, emb) in enumerate(zip(centerlines, embeddings), start=1):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)